elk/training/ccs_reporter.py

```diff
@@ -88,6 +88,8 @@ def __init__(
         num_variants: int = 1,
     ):
         super().__init__()
+        self._is_training = True
```

Comment:
if self._is_training is false during the call to platt_scale, then it won't work because the platt scaling parameter needs to be used for it to be updated in the backwards pass

Reply:
Does it not?
```python
if self._is_training:
    return raw_scores
else:
    platt_scaled_scores = raw_scores.mul(self.scale).add(self.bias).squeeze(-1)
    return platt_scaled_scores
```

Anyway seems simple to test by comparing the results with and without the platt scaling params added